products/BellHybrid/apps/application-bell-settings/windows/BellSettingsWindow.cpp

```diff
@@ -44,6 +44,7 @@ namespace gui
 
         addWinSettings(utils::translate("app_bell_settings_alarm_settings"), BellSettingsAlarmSettingsMenuWindow::name);
         addWinSettings(utils::translate("app_bell_settings_bedtime_tone"), window::name::bellSettingsBedtimeTone);
+        addWinSettings(utils::translate("app_bell_settings_advanced_time_units"), window::name::bellSettingsTimeUnits);
```

Comment:
its just a minor, but maybe if we are moving time&units from advanced, then we should also rename:
`app_bell_settings_advanced_time_units`
to 
`app_bell_setting_time_units`?

Reply:
Changed